src/main/java/seedu/address/model/person/Person.java

```diff
@@ -23,10 +24,24 @@ public class Person {
     // Data fields
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
+    private LocalDateTime time = null;
```

Comment:
Will it be better if you insert a default date, instead of using Null to avoid a null pointer exception?